specs/subresourceintegrity/spec.markdown

```diff
@@ -318,7 +330,21 @@ resources accessed over a `file` scheme URL are unlikely to be
 eligible for integrity checks.
 {:.note}
 
+One should note that being a [privileged context][] (e.g. HTTPS) is not
+necessary for the use of integrity validation. Because resource integrity is
+only an application level security tool, and it does not change the security
+state of the user agent, a privileged context is unnecessary. However, if
+integrity is used in an [unprivileged context][] (e.g. HTTP), users should be
+aware that the integrity provides <em>no security guarantees</em> at all. Even
+if the integrity is used in a privileged context, but on a resource that is not
+potentially secure, then privacy is at great risk. For these reasons, authors
```

Comment:
this will be blocked by mixed content anyhow, so I am not sure you need to say this.